vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -630,6 +646,56 @@ supported:
         </Property>
       </ComplexType>
 
+      <Term Name="ReadRestrictions" Type="Capabilities.ReadRestrictionType" Nullable="false" AppliesTo="EntitySet Singleton">

```

Comment:
I still have **ReadRestrictions** here since I was hoping to discuss this a bit more before taking it out completely.